src/shared/components/ui/post-card.tsx

```
@@ -111,16 +79,25 @@ export default function PostCard({
             })}
           </span>
           <div className="flex items-center gap-3">
-            <span className="flex items-center gap-1">
-              <Heart className="size-3.5" />
+            <span
+              className="flex items-center gap-1"
+              aria-label={`좋아요 ${likeCount}개`}
+            >
```

Biome (2.4.6)

[error] The ARIA attribute 'aria-label' is not supported by this element.

(lint/a11y/useAriaPropsSupportedByRole)